.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,21 @@
+name: SwiftLint
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
+
+jobs:
+  SwiftLint:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: GitHub Action for SwiftLint
+      if: github.event.pull_request.base.repo.id == github.event.pull_request.head.repo.id
```

Comment:
Maybe a comment explaining why this is needed?

Reply:
I added comment